samples/client/petstore/java/jersey2-java6/src/main/java/io/swagger/client/model/AdditionalPropertiesClass.java

```diff
@@ -59,7 +47,7 @@ public AdditionalPropertiesClass putMapPropertyItem(String key, String mapProper
    * Get mapProperty
    * @return mapProperty
   **/
-  @ApiModelProperty(example = "null", value = "")
+  @ApiModelProperty(value = "")
```

Comment:
It looks like previously we got a string `"null"` instead of a real `null` for the example, which confused the null-check.
I didn't figure out which commit/PR fixed this.